### ISSUE TITLE
Question about provided pretrained weights "vgg16_deeplab_aspp.npy"

### ISSUE THREAD

**Issue by terenceylchow124** [xtudbxk/DSRG-tensorflow#29]
Hi, Thank you for preparing the tensorflow DSRG model. In your mark down, it was mentioned that there is the pretrained model which can be downloaded and loaded into the DSRG model. However, I am wondering is this model pretrained by imagenet (for normal classification task) or by Voc2012 (for semantic segmentation). I am not quite sure about the supervision, as the DSRG is the model designed for weakly supervised segmentation, **I just want to make sure how much supervision provided by the pretrained weight.** . Thanks.

**Comment by xtudbxk** [comment]
The model is pretrained on imagenet. And I just converts the pretrained model of deeplab from caffe to tensorflow.

**Comment by terenceylchow124** [comment]
Thanks for your reply!

**Comment by terenceylchow124** [comment]
![Capture](https://user-images.githubusercontent.com/68972473/99312051-79606f80-282b-11eb-99ba-3f985347545d.JPG)
Sorry for  interrupting again, however I am wondering if you have any experience for handling this kind of error. I am trying modify the DSRG model with building some external layers, leading the multiple outputs (two) of the model. And compute the loss like, loss = loss_s + loss_c + loss_e (external loss). Do I need to set up the anything other than building the layers, and construct the loss.

**Comment by xtudbxk** [comment]
It seems the 'pyfunc' cuts off the flow of backward gradients. Just remember that the 'pyfunc' cann't pass the gradients back. So I doubt that there is least one loss function whose all inputs are built upon 'pyfunc' functions.
